tests/test_0692_fsspec.py

```diff
@@ -0,0 +1,75 @@
+# BSD 3-Clause License; see https://github.com/scikit-hep/uproot4/blob/main/LICENSE
+
+import pytest
+
+import uproot
+import uproot.source.fsspec
+
+
+def validate_coffea_file_0(file):
+    data = file["Events/MET_pt"].array(library="np")
+    assert len(data) == 40
```

Comment:
In principle, performing repetitive actions in a helper function would be good practice, but it does make errors in the pytest output harder to read.

You've probably noticed that the other tests do a lot of copy-paste. We have different standards for the testing code: it doesn't need to be as nice and maintainable as the package code—most of it is never changed after first being written (which is why we name test files after PR numbers, as I see you've noticed).

Reply:
Makes sense!